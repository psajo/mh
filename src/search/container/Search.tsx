import { Col, Row, Typography } from 'antd';
import 'antd/dist/antd.css';
import React from 'react';
import Settings from '../component/Settings';
import SearchInput from './SearchInput';

export default function Search(): JSX.Element {
  return (
    <>
      <Row justify="end" style={{ padding: 20 }}>
        <Col>
          <Settings logout={() => {}} />
        </Col>
      </Row>
      <Row justify="center" style={{ marginTop: 100 }}>
        <Col>
          <Typography.Title style={{ fontFamily: 'Caligraphy' }}>
            찾 아 야 한 다
          </Typography.Title>
        </Col>
      </Row>
      <Row justify="center" style={{ marginTop: 50 }}>
        <Col span={12}>
          <SearchInput />
        </Col>
      </Row>
    </>
  );
}
