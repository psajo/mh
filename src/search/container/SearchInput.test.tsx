import { render } from '@testing-library/react';
import React from 'react';
import Search from './Search';

jest.mock('../component/Settings', () => () => <></>);
jest.mock('./SearchInput', () => () => <></>);
test('shoud match the snapshot when render Search Component', () => {
  //given

  //when
  const component = render(<Search />);
  //then
  expect(component).toMatchSnapshot();
});
