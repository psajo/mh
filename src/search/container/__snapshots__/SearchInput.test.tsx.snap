// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shoud match the snapshot when render Search Component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-row ant-row-end"
        style="padding: 20px;"
      >
        <div
          class="ant-col"
        />
      </div>
      <div
        class="ant-row ant-row-center"
        style="margin-top: 100px;"
      >
        <div
          class="ant-col"
        >
          <h1
            class="ant-typography"
            style="font-family: Caligraphy;"
          >
            찾 아 야 한 다
          </h1>
        </div>
      </div>
      <div
        class="ant-row ant-row-center"
        style="margin-top: 50px;"
      >
        <div
          class="ant-col ant-col-12"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-row ant-row-end"
      style="padding: 20px;"
    >
      <div
        class="ant-col"
      />
    </div>
    <div
      class="ant-row ant-row-center"
      style="margin-top: 100px;"
    >
      <div
        class="ant-col"
      >
        <h1
          class="ant-typography"
          style="font-family: Caligraphy;"
        >
          찾 아 야 한 다
        </h1>
      </div>
    </div>
    <div
      class="ant-row ant-row-center"
      style="margin-top: 50px;"
    >
      <div
        class="ant-col ant-col-12"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
